commons/src/testing.rs

```diff
@@ -3,9 +3,15 @@
 use failure::Fallible;
 use tokio::runtime::current_thread::Runtime;
 
+/// Initialize logging.
+pub fn init_logger() -> Fallible<()> {
+    env_logger::try_init_from_env(env_logger::Env::default())?;
```

Comment:
The original location where you moved this from didn't have the error unwrapping. I think the `?` in here will make this failing as soon as there two tests using it, as the logger for subsequent tests is already initialized by the first running test.

Reply:
That's right, I didn't want to swallow the error here. The caller can if it wants to. Do you prefer an opinionated abstraction here?

Reply:
If that was on purpose and we plan to stick with swallowing error at caller site, I'm with keeping this.